lib/view_model/contact_list/contact_list_view_model.dart

```diff
@@ -44,4 +45,20 @@ abstract class ContactListViewModelBase with Store {
   StreamSubscription<BoxEvent>? _subscription;
 
   Future<void> delete(ContactRecord contact) async => contact.original.delete();
+
+  List<ContactRecord> getContacts(bool isEditable, CryptoCurrency? cur) {
+    if (!isEditable) {
+      return contacts.where((element) => element.type == cur).toList();
+    }
+
+    return contacts;
+  }
+
+  List<WalletContact> getWallets(bool isEditable, CryptoCurrency? cur) {
+    if (!isEditable) {
+      return walletContacts.where((element) => element.type == cur).toList();
+    }
+
+    return walletContacts;
+  }
```

Comment:
Could be here case when it's `isEditable` but we need to get only special currency filtered contacts ? Or maybe make sense to just check for is `null` for `cur` here ?

Reply:
I also removed  `isEditable`  for contact_list_page.dart